### ISSUE TITLE
Weighted assignment affecting grade calculator

### ISSUE THREAD

**Issue by Aqtion** [QLSCO/ProjectAlpha#43]
**Is your feature request related to a problem? Please describe.**
Finding final grade calculators on the internet isn't hard, but when it comes to finding calculators that determine how weighted assignments affect your grade, options are scarce.

**Describe the solution you'd like**
A simple add-on to the gpa calculator we already have, just further expanding our reach as a grade calculator type website.

**Comment by Aqtion** [comment]
i am so cool.

**Comment by Aqtion** [comment]
implementing logic shouldn't be hard, maybe a subsection to this gpac section of the site

**Comment by StephenXie** [comment]
Yes, the logic of this calculator should be fairly straightforward.

N = Number of graded assignments
A = Average assignment grade
W = Weightage in decimal
S = Score for Assignment

Only one assignment(e.g. final exam, final project, etc.)
Final Grade = Initial Grade*(1-W) + S*W

Multiple assignments
Final Grade = Initial Grade*(1-W) + ((A*N+S)/(N+1))*W
